posthog/temporal/tests/data_imports/test_postgres_source.py

```diff
@@ -24,9 +24,42 @@
 POSTGRES_TABLE_NAME = "test_table"
 
 TEST_DATA = [
-    (1, "John Doe", "john@example.com", dt.datetime(2025, 1, 1, tzinfo=dt.UTC), 100),
-    (2, "Jane Smith", "jane@example.com", dt.datetime(2025, 1, 2, tzinfo=dt.UTC), 2000000),
-    (3, "Bob Wilson", "bob@example.com", dt.datetime(2025, 1, 3, tzinfo=dt.UTC), 3409892966),
+    (
+        1,
+        "John Doe",
+        "john@example.com",
+        dt.datetime(2025, 1, 1, tzinfo=dt.UTC),
+        100,
+        "[0,2)",
+        '["2025-05-20 00:00:00+00","2025-05-20 01:00:00+00")',
+    ),
+    (
+        2,
+        "Jane Smith",
+        "jane@example.com",
+        dt.datetime(2025, 1, 2, tzinfo=dt.UTC),
+        2000000,
+        "[2,4)",
+        '["2025-05-20 00:00:00+00","2025-05-20 01:00:00+00")',
+    ),
+    (
+        3,
+        "Bob Wilson",
+        "bob@example.com",
+        dt.datetime(2025, 1, 3, tzinfo=dt.UTC),
+        3409892966,
+        "[4,6)",
+        '["2025-05-20 00:00:00+00","2025-05-20 01:00:00+00")',
+    ),
+    (
+        4,
+        "Wob Bilson",
+        "wob@example.com",
+        dt.datetime(2025, 1, 3, tzinfo=dt.UTC),
+        4,
+        "[5,7)",
+        None,
+    ),
```

Comment:
Perhaps worth including a range with negative numbers to test the signed integer and floating point number support as well?

Reply:
Sure, I added a few negatives, and a numeric range. Test is still passing with new data.